opentelemetry-otlp/tests/integration_test/actual/.gitignore

```diff
@@ -0,0 +1 @@
+*.json
```

Comment:
not sure if there are best practices around maintaining multiple gitignore files? maybe move them all to the root one itself.